ophyd/epics_motor.py

```diff
@@ -155,7 +259,17 @@ def _move_changed(self, timestamp=None, value=None, sub_type=None,
                            value=value, **kwargs)
 
         if was_moving and not self._moving:
-            self._done_moving(timestamp=timestamp, value=value)
+            success = True
+            # Check if we are moving towards the low limit switch
+            if self.motor_direction_of_travel.get() == 0:
+                if self.at_low_limit_switch:
+                    success = False
+            # No, we are going to the high limit switch
+            else:
+                if self.at_high_limit_switch:
+                    success = False
```

Comment:
Yes, it is. Imagine the following situation: 
If the user try to send a movement in the opposite direction of the limit switch which is active but the distance wasn't enough to deactivate it we need to consider this a valid movement. That's why the direction of travel is being considered! ;)